tests/scripts/expect/cli-tcat-advertisement.exp

```diff
@@ -32,44 +32,64 @@ source "tests/scripts/expect/_common.exp"
 spawn_node 1 "cli"
 
 switch_node 1
-send "tcat vendorid ianapen f378\n"
+send "tcat advid ianapen f378\n"
 expect_line "Done"
 
-send "tcat vendorid\n"
+send "tcat advid\n"
 expect_line "type ianapen, value: f378"
 expect_line "Done"
 
-send "tcat vendorid ianapen f378aabb\n"
+send "tcat advid ianapen f378aabb\n"
 expect_line "Done"
 
-send "tcat vendorid\n"
+send "tcat advid\n"
 expect_line "type ianapen, value: f378aabb"
 expect_line "Done"
 
-send "tcat vendorid oui24 f378aa\n"
+send "tcat advid oui24 f378aa\n"
 expect_line "Done"
 
-send "tcat vendorid\n"
+send "tcat advid\n"
 expect_line "type oui24, value: f378aa"
 expect_line "Done"
 
-send "tcat vendorid oui36 f378aabbcc\n"
+send "tcat advid oui36 f378aabbcc\n"
 expect_line "Done"
 
-send "tcat vendorid\n"
+send "tcat advid\n"
 expect_line "type oui36, value: f378aabbcc"
 expect_line "Done"
 
-send "tcat vendorid discriminator f378aabbdd\n"
+send "tcat advid discriminator f378aabbdd\n"
 expect_line "Done"
 
-send "tcat vendorid\n"
+send "tcat advid\n"
 expect_line "type discriminator, value: f378aabbdd"
 expect_line "Done"
 
-send "tcat vendorid empty\n"
+send "tcat advid empty\n"
 expect_line "Done"
 
-send "tcat vendorid\n"
+send "tcat advid\n"
 expect_line "empty"
 expect_line "Done"
+
+send "tcat devid\n"
+expect_line "empty"
+expect_line "Done"
+
+send "tcat devid aaaa\n"
+expect_line "aaaa"
+expect_line "Done"
+
+send "tcat devid\n"
+expect_line "aaaa"
+expect_line "Done"
+
+send "tcat devid empty\n"
+expect_line "empty"
```

Comment:
as output of a command `tcat devid empty` or better `tcat devid clear` I would expect only "Done" as result?  Since it's a command to clear any IDs, not list them.

Reply:
fixed.